vector fails to build on amd64

The package vector with version 0.47.0 and epoch 0 fails to build from source on amd64 with the following error:

```
2025-06-09 08:23:16.230302611 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/include/gssrpc/rename.h:106:33: note: 'gssrpc_xdr_authgssapi_creds' declared here
2025-06-09 08:23:16.230312223 +0000 UTC     106 | #define xdr_authgssapi_creds    gssrpc_xdr_authgssapi_creds
2025-06-09 08:23:16.230318418 +0000 UTC         |                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~
2025-06-09 08:23:16.230322823 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/include/gssrpc/auth_gssapi.h:79:8: note: in expansion of macro 'xdr_authgssapi_creds'
2025-06-09 08:23:16.230327595 +0000 UTC      79 | bool_t xdr_authgssapi_creds(XDR *, auth_gssapi_creds *);
2025-06-09 08:23:16.230332292 +0000 UTC         |        ^~~~~~~~~~~~~~~~~~~~
2025-06-09 08:23:16.230336813 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/include/gssrpc/xdr.h:105:18: note: 'xdrproc_t' declared here
2025-06-09 08:23:16.230341848 +0000 UTC     105 | typedef bool_t (*xdrproc_t)();
2025-06-09 08:23:16.230346582 +0000 UTC         |                  ^~~~~~~~~
2025-06-09 08:23:16.23035149 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/include/gssrpc/rename.h:106:33: error: passing argument 1 of 'gssrpc_xdr_free' from incompatible pointer type [-Wincompatible-pointer-types]
2025-06-09 08:23:16.230361843 +0000 UTC     106 | #define xdr_authgssapi_creds    gssrpc_xdr_authgssapi_creds
2025-06-09 08:23:16.230368378 +0000 UTC         |                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~
2025-06-09 08:23:16.230373585 +0000 UTC         |                                 |
2025-06-09 08:23:16.230377858 +0000 UTC         |                                 int (*)(XDR *, auth_gssapi_creds *) {aka int (*)(XDR *, struct _auth_gssapi_creds *)}
2025-06-09 08:23:16.230386628 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/lib/rpc/svc_auth_gssapi.c:654:20: note: in expansion of macro 'xdr_authgssapi_creds'
2025-06-09 08:23:16.230392403 +0000 UTC     654 |           xdr_free(xdr_authgssapi_creds, &creds);
2025-06-09 08:23:16.230397098 +0000 UTC         |                    ^~~~~~~~~~~~~~~~~~~~
2025-06-09 08:23:16.230401558 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/include/gssrpc/xdr.h:335:27: note: expected 'xdrproc_t' {aka 'int (*)(void)'} but argument is of type 'int (*)(XDR *, auth_gssapi_creds *)' {aka 'int (*)(XDR *, struct _auth_gssapi_creds *)'}
2025-06-09 08:23:16.230406538 +0000 UTC     335 | extern void     xdr_free (xdrproc_t, void *);
2025-06-09 08:23:16.230411126 +0000 UTC         |                           ^~~~~~~~~
2025-06-09 08:23:16.230415842 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/include/gssrpc/rename.h:106:33: note: 'gssrpc_xdr_authgssapi_creds' declared here
2025-06-09 08:23:16.230421738 +0000 UTC     106 | #define xdr_authgssapi_creds    gssrpc_xdr_authgssapi_creds
2025-06-09 08:23:16.230426775 +0000 UTC         |                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~
2025-06-09 08:23:16.23043718 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/include/gssrpc/auth_gssapi.h:79:8: note: in expansion of macro 'xdr_authgssapi_creds'
2025-06-09 08:23:16.230442668 +0000 UTC      79 | bool_t xdr_authgssapi_creds(XDR *, auth_gssapi_creds *);
2025-06-09 08:23:16.230447736 +0000 UTC         |        ^~~~~~~~~~~~~~~~~~~~
2025-06-09 08:23:16.23045865 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/include/gssrpc/xdr.h:105:18: note: 'xdrproc_t' declared here
2025-06-09 08:23:16.230464086 +0000 UTC     105 | typedef bool_t (*xdrproc_t)();
2025-06-09 08:23:16.230468731 +0000 UTC         |                  ^~~~~~~~~
2025-06-09 08:23:16.23047324 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/lib/rpc/pmap_rmt.c: In function 'gssrpc_clnt_broadcast':
2025-06-09 08:23:16.230478378 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/lib/rpc/pmap_rmt.c:346:49: error: assignment to 'xdrproc_t' {aka 'int (*)(void)'} from incompatible pointer type 'int (*)(XDR *, struct rmtcallres *)' [-Wincompatible-pointer-types]
2025-06-09 08:23:16.230483413 +0000 UTC     346 |                 msg.acpted_rply.ar_results.proc = xdr_rmtcallres;
2025-06-09 08:23:16.230488177 +0000 UTC         |                                                 ^
2025-06-09 08:23:16.230493614 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/include/gssrpc/rename.h:167:33: note: 'gssrpc_xdr_rmtcallres' declared here
2025-06-09 08:23:16.230499128 +0000 UTC     167 | #define xdr_rmtcallres          gssrpc_xdr_rmtcallres
2025-06-09 08:23:16.230504518 +0000 UTC         |                                 ^~~~~~~~~~~~~~~~~~~~~
2025-06-09 08:23:16.230509686 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/lib/rpc/pmap_rmt.c:156:1: note: in expansion of macro 'xdr_rmtcallres'
2025-06-09 08:23:16.230526808 +0000 UTC     156 | xdr_rmtcallres(
2025-06-09 08:23:16.230532828 +0000 UTC         | ^~~~~~~~~~~~~~
2025-06-09 08:23:16.23053866 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/include/gssrpc/xdr.h:105:18: note: 'xdrproc_t' declared here
2025-06-09 08:23:16.230547652 +0000 UTC     105 | typedef bool_t (*xdrproc_t)();
2025-06-09 08:23:16.230553346 +0000 UTC         |                  ^~~~~~~~~
2025-06-09 08:23:16.23055835 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/lib/rpc/pmap_rmt.c:399:49: error: assignment to 'xdrproc_t' {aka 'int (*)(void)'} from incompatible pointer type 'int (*)(XDR *, void *)' [-Wincompatible-pointer-types]
2025-06-09 08:23:16.230563718 +0000 UTC     399 |                 msg.acpted_rply.ar_results.proc = xdr_void;
2025-06-09 08:23:16.230572506 +0000 UTC         |                                                 ^
2025-06-09 08:23:16.230578247 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/include/gssrpc/rename.h:246:25: note: 'gssrpc_xdr_void' declared here
2025-06-09 08:23:16.230588648 +0000 UTC     246 | #define xdr_void        gssrpc_xdr_void
2025-06-09 08:23:16.230594518 +0000 UTC         |                         ^~~~~~~~~~~~~~~
2025-06-09 08:23:16.230599298 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/include/gssrpc/xdr.h:251:17: note: in expansion of macro 'xdr_void'
2025-06-09 08:23:16.230604353 +0000 UTC     251 | extern bool_t   xdr_void(XDR *, void *);
2025-06-09 08:23:16.230609732 +0000 UTC         |                 ^~~~~~~~
2025-06-09 08:23:16.230614218 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/include/gssrpc/xdr.h:105:18: note: 'xdrproc_t' declared here
2025-06-09 08:23:16.230618988 +0000 UTC     105 | typedef bool_t (*xdrproc_t)();
2025-06-09 08:23:16.23062419 +0000 UTC         |                  ^~~~~~~~~
2025-06-09 08:23:16.23063386 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/lib/rpc/pmap_rmt.c:401:24: error: too many arguments to function 'xresults'; expected 0, have 2
2025-06-09 08:23:16.230640194 +0000 UTC     401 |                 (void)(*xresults)(xdrs, resultsp);
2025-06-09 08:23:16.230644526 +0000 UTC         |                       ~^~~~~~~~~~ ~~~~
2025-06-09 08:23:16.230649001 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/lib/rpc/svc_auth_gssapi.c: In function 'svc_auth_gssapi_wrap':
2025-06-09 08:23:16.230654015 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/lib/rpc/svc_auth_gssapi.c:1089:19: error: too many arguments to function 'xdr_func'; expected 0, have 2
2025-06-09 08:23:16.230660218 +0000 UTC    1089 |           return (*xdr_func)(out_xdrs, xdr_ptr);
2025-06-09 08:23:16.230664778 +0000 UTC         |                  ~^~~~~~~~~~ ~~~~~~~~
2025-06-09 08:23:16.23066953 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/lib/rpc/svc_auth_gssapi.c: In function 'svc_auth_gssapi_unwrap':
2025-06-09 08:23:16.230674803 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/lib/rpc/svc_auth_gssapi.c:1113:19: error: too many arguments to function 'xdr_func'; expected 0, have 2
2025-06-09 08:23:16.23067995 +0000 UTC    1113 |           return (*xdr_func)(in_xdrs, (auth_gssapi_init_arg *)(void *) xdr_ptr);
2025-06-09 08:23:16.23068482 +0000 UTC         |                  ~^~~~~~~~~~ ~~~~~~~
2025-06-09 08:23:16.230689128 +0000 UTC   make[2]: *** [Makefile:500: svc_auth_gss.o] Error 1
2025-06-09 08:23:16.230698371 +0000 UTC   make[2]: *** [Makefile:500: svc_udp.o] Error 1
2025-06-09 08:23:16.230703743 +0000 UTC   make[2]: *** [Makefile:500: pmap_rmt.o] Error 1
2025-06-09 08:23:16.230709708 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/lib/rpc/svc_tcp.c: In function 'svctcp_freeargs':
2025-06-09 08:23:16.230714618 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/lib/rpc/svc_tcp.c:470:18: error: too many arguments to function 'xdr_args'; expected 0, have 2
2025-06-09 08:23:16.230719154 +0000 UTC     470 |         return ((*xdr_args)(xdrs, args_ptr));
2025-06-09 08:23:16.230724408 +0000 UTC         |                 ~^~~~~~~~~~ ~~~~
2025-06-09 08:23:16.230729895 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/lib/rpc/svc_tcp.c: In function 'svctcp_reply':
2025-06-09 08:23:16.230747997 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/lib/rpc/svc_tcp.c:491:50: error: assignment to 'xdrproc_t' {aka 'int (*)(void)'} from incompatible pointer type 'int (*)(XDR *, void *)' [-Wincompatible-pointer-types]
2025-06-09 08:23:16.230757823 +0000 UTC     491 |                 msg->acpted_rply.ar_results.proc = xdr_void;
2025-06-09 08:23:16.23076372 +0000 UTC         |                                                  ^
2025-06-09 08:23:16.230770315 +0000 UTC   In file included from ../../include/gssrpc/types.h:128,
2025-06-09 08:23:16.230780246 +0000 UTC                    from /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/include/gssrpc/rpc.h:42,
2025-06-09 08:23:16.230786414 +0000 UTC                    from /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/lib/rpc/svc_tcp.c:48:
2025-06-09 08:23:16.230792084 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/include/gssrpc/rename.h:246:25: note: 'gssrpc_xdr_void' declared here
2025-06-09 08:23:16.23079721 +0000 UTC     246 | #define xdr_void        gssrpc_xdr_void
2025-06-09 08:23:16.230801915 +0000 UTC         |                         ^~~~~~~~~~~~~~~
2025-06-09 08:23:16.230808578 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/include/gssrpc/xdr.h:251:17: note: in expansion of macro 'xdr_void'
2025-06-09 08:23:16.230836574 +0000 UTC     251 | extern bool_t   xdr_void(XDR *, void *);
2025-06-09 08:23:16.230842168 +0000 UTC         |                 ^~~~~~~~
2025-06-09 08:23:16.23084726 +0000 UTC   In file included from /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/include/gssrpc/rpc.h:46:
2025-06-09 08:23:16.230852688 +0000 UTC   /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/krb5/src/include/gssrpc/xdr.h:105:18: note: 'xdrproc_t' declared here
2025-06-09 08:23:16.230858323 +0000 UTC     105 | typedef bool_t (*xdrproc_t)();
2025-06-09 08:23:16.230863678 +0000 UTC         |                  ^~~~~~~~~
2025-06-09 08:23:16.230868433 +0000 UTC   make[2]: *** [Makefile:500: svc_auth_gssapi.o] Error 1
2025-06-09 08:23:16.230874583 +0000 UTC   make[2]: *** [Makefile:500: svc_tcp.o] Error 1
2025-06-09 08:23:16.230882976 +0000 UTC   make[1]: *** [Makefile:1005: all-recurse] Error 1
2025-06-09 08:23:16.230889533 +0000 UTC 
2025-06-09 08:23:16.230894657 +0000 UTC   thread 'main' panicked at /home/build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/krb5-src-0.3.2+1.19.2/build.rs:138:41:
2025-06-09 08:23:16.230904553 +0000 UTC   make failed in lib
2025-06-09 08:23:16.230911175 +0000 UTC   note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
2025-06-09 08:23:16.299641494 +0000 UTC make: *** [Makefile:195: build] Error 101
2025-06-09 08:23:23.715416772 +0000 UTC failed to build package: unable to run package vector pipeline: unable to run pipeline: exit status 2
2025-06-09 08:23:23.758308803 +0000 UTC failed to build: command melange failed: exit status 1
2025-06-09 08:23:23.80599607 +0000 UTC build failed: command melange failed: exit status 1
```

--- a/vector.yaml
+++ b/vector.yaml
@@ -1,7 +1,7 @@
 package:
   name: vector
   version: "0.47.0"
-  epoch: 0
+  epoch: 1
   description: End-to-end observability data pipeline
   copyright:
     - license: MPL-2.0
@@ -15,6 +15,7 @@ environment:
       - bash
       - build-base
       - cyrus-sasl-dev
+      - gcc-14-default
       - librdkafka
       - librdkafka-dev
       - openssl

